Zotero/Scenes/Detail/PDF/Views/PDFReaderViewController.swift

```diff
@@ -487,12 +493,17 @@ class PDFReaderViewController: UIViewController {
             // Move to closest point. Use different threshold for vertical/horizontal rotation.
             let threshold: CGFloat
             if frame.height > frame.width {
-                threshold = self.isCompactWidth ? 150 : 100
+                threshold = self.isCompactWidth ? 400 : 350
```

Comment:
I chose these numbers by testing and seeing what worked better for both the iPad screen and iPhone screen.

Reply:
@angiemugo I don't think these numbers work better than before. Here you can see it immediately wants to push the toolbar up, which doesn't feel right:
https://www.dropbox.com/s/0fghix5v7dnc1tr/RPReplay_Final1682584450.MP4?dl=0

You can maybe make it proportional to current screen height, something like `UIScreen.main.frame.height / 4` to make the threshold the top quarter of screen.

@dstillman what do you think?

Reply:
Are we basing this on the frame borders or the finger position? I think it should be the latter. I wouldn't expect it to pin to the top until my finger was over the top toolbar zone.

Reply:
@dstillman it should be center of the toolbar. Initially it was set up so that when the toolbar center is above some threshold it moves up, moving the finger seemed too much, but it can be changed easily.

Reply:
It was fine before when there were only three positions, but with the top position now, I think it's going to feel weird to have it jump to the top for no apparent reason. I think PDF Viewer is doing finger.

Reply:
@dstillman that's why I don't like the threshold change to around 400 px from top. When it was 100 px you'd have to move the toolbar a bit higher before it wants to move up. I'd add some distance between `pinned` and `top` and keep the center point instead of finger point. PDF Viewer does finger that's true, but to me it seems a bit too far. I'm not against either though.

Reply:
I revoked the changes on the threshold and instead implemented some changes to the condition for pinned/top position. While earlier I was checking where the minY of the annotation bar was to determine the position, I am now working with centerY. 
The reason I had changed the threshold was that while working with minY, by the time we met the threshold conditions, the pinned condition is what would always be met.